core/ledger/pvtdatastorage/kv_encoding.go

```diff
@@ -302,6 +302,16 @@ func encodePurgeMarkerKey(k *purgeMarkerKey) []byte {
 	return encKey
 }
 
+func rangeScanKeysForPurgeMarkers() ([]byte, []byte) {
+	return purgeMarkerKeyPrefix, []byte{purgeMarkerKeyPrefix[0] + 1}
+}
+
+func driveHashedIndexKeyRangeFromPurgeMarker(purgeMarkerKey, purgeMarkerVal []byte) ([]byte, []byte) {
```

Comment:
Add a function comment explaining that "purgeMarkerVal" represents the purged height, and this function will return a startKey and endKey that will serve to process and remove all HashedIndex entries from genesis height (startKey) up until the purged height (endKey).

Reply:
Done.